frontend/src/components/PsyListing/PsyTable.jsx

```diff
@@ -114,13 +114,14 @@ const PsyTable = ({
                 >
                   <Icon name="ri-phone-fill" size="2x" />
                 </div>
-                <Button
+                <Link
                   data-test-id="psy-table-row-profil-button"
-                  secondary
+                  href={`/trouver-un-psychologue/${psychologist.dossierNumber}`}
                   onClick={() => goToProfile(psychologist)}
```

Comment:
OnClick should not be needed anymore ?

Reply:
Yes I need it to do the "history trick" and come back to the psy list with previous filter. (I saw that with our beautiful integration tests :))